integration_test/ops_agent_test.go

```diff
@@ -3227,10 +3234,17 @@ func testPrometheusMetrics(t *testing.T, opsAgentConfig string, testChecks []moc
 			t.Fatal(err)
 		}
 
-		// 2. Setup the golang and start the go http server
+		// 2. Setup the golang
 		if err := installGolang(ctx, logger, vm); err != nil {
 			t.Fatal(err)
 		}
+
+		// 3. Build the http server
+		if err := buildGoBinary(ctx, logger, vm, serviceFiles[0].remote, remoteWorkDir); err != nil {
```

Comment:
We will need to update `integration_test/testdata/prometheus/setup_json_exporter.sh` as well with this build step. This `setup_json_exporter.sh` is used by `TestPrometheusMetricsWithJSONExporter` and it also uses the `http-server-for-prometheus-test` service.

Reply:
Done! I think I just need to update the setup step.